src/Controller/SubsiteXHRController.php

```diff
@@ -36,33 +43,21 @@ public function canView($member = null)
     }
 
     /**
-     * Allow access if user allowed into the CMS at all.
+     * Get a Pjax response negotiator for the subsite list
      */
-    public function canAccess()
```

Comment:
`canAccess()` is a non-standard method that's only applied to `LeftAndMain` classes via the `LeftAndMainSubsites` extension. It no longer applies here, so we use `required_permission_codes` instead set to the less restrictive of these since `canView()` is handling the main access check for us.